tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -2198,8 +2165,8 @@ void SwiftLangSupport::getCursorInfo(
 }
 
 void SwiftLangSupport::getDiagnostics(
```

Comment:
I think I purposefully left `getDiagnostics` with just `PrimaryFilePath`, since I expected the diagnostics in any buffers underneath to be part of the diagnostics there. But I suppose this doesn't hurt either way, would certainly make it easier to get diagnostics for just the single buffer.

Assuming it actually works, probably worth adding a test for that case.

Reply:
I removed the change from this PR. I’ll test if it works. If it does, I’ll open another PR. If it doesn’t 🤷🏽

Reply:
Just checked and we don’t get any diagnostics if we pass in an input buffer name. Since we don’t have a need for it at the moment, I’ll leave it as-is for now.

Reply:
Sounds good, thanks.